roles/core/node/tasks/install_dir_pkg.yml

```diff
@@ -0,0 +1,201 @@
+---
+# directory package installation method
+
+- block:  ## run_once: true
+    - name: install | Stat directory installation package
+      stat:
+        path: "{{ scale_install_directory_pkg_path }}"
+      register: scale_install_dirpkg
+
+    - name: install | Check directory installation package
+      assert:
+        that: scale_install_dirpkg.stat.exists
+        msg: >-
+          Please set the variable 'scale_install_directory_pkg_path' to point to the
+          local installation package (accessible on Ansible control machine)!
+  run_once: true
+  delegate_to: localhost
+
+
+- name: install| Creates default directory
+  file:
+    path: "{{ gpfs_extracted_path }}"
+    state: directory
+    mode: a+x
+    recurse: yes
+
+- name: install | Stat extracted packages
+  stat:
+    path: "{{ gpfs_extracted_path + '/' + scale_install_directory_pkg_path | basename }}"
+  register: scale_install_gpfs_rpmdir
+
+#
+# Copy installation directory package
+#
+
+- block:  ## when: not scale_install_gpfs_rpmdir.stat.exists
+    - name: install | Stat default directory
+      stat:
+        path: "{{ gpfs_extracted_path }}"
+      register: scale_install_localpkg_tmpdir
+
+    - name: install | Check default directory
+      assert:
+        that:
+          - scale_install_localpkg_tmpdir.stat.exists
+          - scale_install_localpkg_tmpdir.stat.isdir
+        msg: >-
+          Please set the variable 'gpfs_extracted_path' to point
+          to a default directory on the remote system!
```

Comment:
Since we have just created the directory gpfs_extracted_path in the task above, there is no need to check if it exists again here.